Rakefile

```diff
@@ -1,6 +1,6 @@
-require "bundler/gem_tasks"
-require "rspec/core/rake_task"
+require 'bundler/gem_tasks'
+require 'rspec/core/rake_task'
```

Comment:
im guessing most of these single quote changes and the link are by rubocop for formatting.